tv_apps/browser/style/toolbar.css

```diff
@@ -741,58 +741,74 @@ input::-moz-focus-inner {
 }
 
 #menu-block li {
-  display: table;
   width: 100%;
-  height: 72px;
-  font-size: 32px;
-  border-bottom: solid 1px #7a7a7a;
-  -moz-transition: height 0.42s cubic-bezier(0.25, 0, 0, 1.0),
-                   border-bottom 0.42s cubic-bezier(0.25, 0, 0, 1.0),
-                   font-size 0.42s cubic-bezier(0.25, 0, 0, 1.0);
+  height: 7.2rem;
+  font-size: 3.2rem;
+  border-bottom: solid 0.1rem #7a7a7a;
+  transition: height 0.42s cubic-bezier(0.25, 0, 0, 1.0),
+              border-bottom 0.42s cubic-bezier(0.25, 0, 0, 1.0);
 }
 #menu-block li:last-child {
   /* border-bottom: 0px; */
 }
 
 #menu-block li:hover {
-  border-bottom: solid 6px #7a7a7a;
-  height: 92px;
+  border-bottom: solid 0.6rem #7a7a7a;
+  height: 9.2rem;
 }
 #menu-block li:hover a {
-  font-size: 38px;
+  font-size: 3.8rem;
+}
+#menu-block li:hover a#pocket-tab:before {
+  background-size: 120% 120%;
+  background-position: center 80%;
 }
 #menu-block li:active a {
   color: #00caf2;
-  padding-left: 20px;
+  padding-left: 2rem;
 }
 
 #menu-block li > a {
-  display: table-cell;
+  display: flex;
   color: #ffffff;
   text-align: left;
-  padding-left: 10px;
-  padding-right: 10px;
-  max-width: 980px;
-  height: 72px;
+  padding-left: 2rem;
+  padding-right: 2rem;
```

Comment:
Same as above, just want to confirm that the change from 10px to 2rem is intended.